tests/Unit/Routes/DataRoutesTest.php

```diff
@@ -37,19 +36,17 @@ public function testVouchersIndexRouteAuthdAdmin()
         $this->actingAs($this->admin, 'admin')
             ->get(route('vouchers.index'))
             ->assertStatus(200)
-            ->assertJsonStructure([ 0 => [
+            ->assertJsonStructure([0 => [
                 'id', 'trader_id', 'code', 'currentstate', 'sponsor_id'
-            ]])
-        ;
+            ]]);
```

Comment:
This is an example of where i would favour the next-line `;` 
Not important though

Reply:
I also favour the next line `;` syntax, but agree it is not important. The reason I bring it up is:

1- Much time has been spent by me and others discussing. @Nikomus points out that there are examples of both styles in this project so it's confusing to know which to go for
2- Some time spend in Code Review looking at the irrelevantly changed lines

Reply:
I don't think it's worth the time to change these back now, but it might be worth agreeing that we do closing `;` on their own line from now on and configuring our editors not to undo this preference in future.

Reply:
I would very much prefer a clear coding standard, especially if there is nothing imposed by the client. I know it would be per project but at least then we're all on the same page

Reply:
Lonely `;` looks weird to me, but more importantly I think it can be demoralising to hold up a PR fiddling with these when it's not a framework convention, or well observed in the project code/defined in docs.